### ISSUE TITLE
Specify Throughput for a Cosmos Container or Database

### ISSUE THREAD

**Issue by Thorstensen** [dotnet/efcore#20983]
When using Entity Framework Core with the `UseCosmos` options, I would like to be able to specify on what level the throughput should be configured. In our current solution, we have a database with several containers. By default, Entity Framework specifies the Throughput(RU/s) on the container during creation of the database. 

I would like an option to specify on what level to configure the throughput(RU/s) on. For instance when using the Action in `UseCosmos`:

**Example 1:**

```
optionsBuilder.UseCosmos(_instanceUrl,  _primaryKey,  _databaseName, options =>
{
    options.ThroughputOn(typeOfThroughput: Level.Database, amount:1000);
});
```
**Example 2:**

```
optionsBuilder.UseCosmos(_instanceUrl,  _primaryKey,  _databaseName, options =>
{
    options.ThroughputOn(typeOfThroughput: Level.Containers, amount:400);
});
```

**Comment by Marusyk** [comment]
+1 it would be a nice fature

**Comment by AndriySvyryd** [comment]
Duplicate of https://github.com/dotnet/efcore/issues/17301

**Comment by ciacco85** [comment]
any update?

**Comment by roji** [comment]
@ciacco85 this issue was closed as a duplicate of #17301, which was done for EF 6.0.
